Akka Cluster errors in logs for non-clustered app

When added Mesmer to an application that doesn't include Akka Cluster, the logs contain this information:
```
[info] 2022-07-28 14:01:50 ERROR i.s.mesmer.extension.AkkaMonitoring  ActorSystem is not properly configured to start cluster monitor of type ClusterEventsMonitor$
[info] 2022-07-28 14:01:50 DEBUG a.a.t.i.r.LocalReceptionist akka://akkaq1-system/system/localReceptionist Actor was registered: ServiceKey[io.scalac.mesmer.core.event.StreamEvent](io.scalac.metric.stream) Actor[akka://akkaq1-system/system/mesmerStreamMonitor/$$b-adapter#533007682]
[info] 2022-07-28 14:01:50 ERROR i.s.mesmer.extension.AkkaMonitoring  ActorSystem is not properly configured to start cluster monitor of type ClusterRegionsMonitorActor$
[info] 2022-07-28 14:01:50 ERROR i.s.mesmer.extension.AkkaMonitoring  ActorSystem is not properly configured to start cluster monitor of type ClusterSelfNodeEventsActor$
```

`sbt new akka/akka-http-quickstart-scala.g8` on run with mesmer agent also throws number of errors

```
[error] Exception in thread "Thread-3" java.lang.NoClassDefFoundError: io/scalac/mesmer/core/util/Retry$
[error]   at io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$.io$scalac$mesmer$otelextension$instrumentations$akka$stream$AkkaStreamMonitorExtension$$registerWhenSystemIsInitialized(AkkaStreamMonitorExtension.scala:84)
[error]   at io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$$anon$1.run(AkkaStreamMonitorExtension.scala:80)
[error]   at java.base/java.lang.Thread.run(Thread.java:1589)
[error] Caused by: java.lang.ClassNotFoundException: io.scalac.mesmer.core.util.Retry$
[error]   at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
[error]   at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
[error]   at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error]   ... 3 more
[error] Exception in thread "main" java.lang.NoClassDefFoundError: akka/cluster/typed/Cluster$
[error]   at io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension$.registerExtension(AkkaClusterMonitorExtension.scala:63)
[error]   at io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension.registerExtension(AkkaClusterMonitorExtension.scala)
[error]   at akka.actor.ActorSystemImpl.<init>(ActorSystem.scala:799)
[error]   at akka.actor.typed.ActorSystem$.createInternal(ActorSystem.scala:289)
[error]   at akka.actor.typed.ActorSystem$.apply(ActorSystem.scala:198)
[error]   at com.example.QuickstartApp$.main(QuickstartApp.scala:40)
[error]   at com.example.QuickstartApp.main(QuickstartApp.scala)
[error] Caused by: java.lang.ClassNotFoundException: akka.cluster.typed.Cluster$
[error]   at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
[error]   at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
[error]   at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error]   ... 7 more
```

This project contains only the following dependencies 
```
      "com.typesafe.akka" %% "akka-http"                % akkaHttpVersion,
      "com.typesafe.akka" %% "akka-http-spray-json"     % akkaHttpVersion,
      "com.typesafe.akka" %% "akka-actor-typed"         % akkaVersion,
      "com.typesafe.akka" %% "akka-stream"              % akkaVersion,
```

--- a/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaActorInstrumentationModule.java
+++ b/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaActorInstrumentationModule.java
@@ -71,27 +71,29 @@ public class MesmerAkkaActorInstrumentationModule extends InstrumentationModule
 
   @Override
   public List<String> getAdditionalHelperClassNames() {
-    return Arrays.asList(
-        "io.scalac.mesmer.core.actor.ActorRefAttributeFactory",
-        "io.scalac.mesmer.core.actor.ConfiguredAttributeFactory",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.Instruments$$anon$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.Instruments",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.Instruments$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.InstrumentsProvider",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.InstrumentsProvider$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.util.EnvelopeContext",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.util.EnvelopeContext$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.impl.otel.ActorCellInstrumentationState",
-        "akka.actor.ProxiedQueue",
-        "akka.actor.BoundedQueueProxy",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtensionId$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension$$anon$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents$ActorCreated",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents$ActorTerminated",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitor",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitor$");
+    return MesmerAkkaHelpers.combine(
+        MesmerAkkaHelpers.coreHelpers(),
+        Arrays.asList(
+            "akka.actor.ProxiedQueue",
+            "akka.actor.BoundedQueueProxy",
+            "io.scalac.mesmer.core.actor.ActorRefAttributeFactory",
+            "io.scalac.mesmer.core.actor.ConfiguredAttributeFactory",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.Instruments$$anon$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.Instruments",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.Instruments$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.InstrumentsProvider",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.InstrumentsProvider$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.util.EnvelopeContext",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.util.EnvelopeContext$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.impl.otel.ActorCellInstrumentationState",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtensionId$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension$$anon$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents$ActorCreated",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents$ActorTerminated",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitor",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitor$"));
   }
 }

--- a/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaClusterInstrumentationModule.java
+++ b/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaClusterInstrumentationModule.java
@@ -39,26 +39,33 @@ public class MesmerAkkaClusterInstrumentationModule extends InstrumentationModul
 
   @Override
   public List<String> getAdditionalHelperClassNames() {
-    return List.of(
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension$$anon$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtensionId$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterEventsMonitor$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterEventsMonitor$MemberEventWrapper",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.OnClusterStartup$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.OnClusterStartup$Initialized",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.OnClusterStartup$Timeout$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterMonitorActor",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command$ClusterMemberEvent",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command$NodeUnreachable",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command$NodeReachable",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.common.SerializableMessage",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$Command",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$Regions",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$$anonfun$$nestedInanonfun$apply$2$1");
+    return MesmerAkkaHelpers.combine(
+        MesmerAkkaHelpers.coreHelpers(),
+        List.of(
+            // Vendor helpers
+            "akka.cluster.typed.Cluster",
+            "akka.cluster.typed.Cluster$",
+            "akka.cluster.typed.internal.AdapterClusterImpl",
+            // Mesmer extension helpers
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtension$$anon$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.AkkaClusterMonitorExtensionId$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterEventsMonitor$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterEventsMonitor$MemberEventWrapper",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.OnClusterStartup$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.OnClusterStartup$Initialized",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.OnClusterStartup$Timeout$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterMonitorActor",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command$ClusterMemberEvent",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command$NodeUnreachable",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterSelfNodeEventsActor$Command$NodeReachable",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.common.SerializableMessage",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$Command",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$Regions",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.cluster.extension.ClusterRegionsMonitorActor$$anonfun$$nestedInanonfun$apply$2$1"));
   }
 }

--- a/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaHelpers.java
+++ b/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaHelpers.java
@@ -1,0 +1,88 @@
+package io.scalac.mesmer.otelextension.akka;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MesmerAkkaHelpers {
+
+  // Mesmer core helpers. With Muzzle plugin this can be safely defined in `isHelperClass`.
+  public static List<String> coreHelpers() {
+    return List.of(
+        "io.scalac.mesmer.core.package$",
+        "io.scalac.mesmer.core.PathMatcher$",
+        "io.scalac.mesmer.core.PathMatcher$Error",
+        "io.scalac.mesmer.core.PathMatcher$Prefix",
+        "io.scalac.mesmer.core.PathMatcher",
+        "io.scalac.mesmer.core.ActorGrouping$",
+        "io.scalac.mesmer.core.ActorGrouping$Error",
+        "io.scalac.mesmer.core.ActorGrouping",
+        "io.scalac.mesmer.core.AkkaDispatcher$",
+        "io.scalac.mesmer.core.ActorPathAttribute",
+        "io.scalac.mesmer.core.Disabled",
+        "io.scalac.mesmer.core.DisabledPath$",
+        "io.scalac.mesmer.core.SomeActorPathAttribute",
+        "io.scalac.mesmer.core.akka.model.AttributeNames$",
+        "io.scalac.mesmer.core.akka.model.package$PushMetrics$",
+        "io.scalac.mesmer.core.akka.stream.BidiFlowForward$",
+        "io.scalac.mesmer.core.akka.stream.BidiFlowForward$$anon$1$$anon$2",
+        "io.scalac.mesmer.core.akka.stream.BidiFlowForward$$anon$1",
+        "io.scalac.mesmer.core.actor.ConfiguredAttributeFactory$$anonfun$1",
+        "io.scalac.mesmer.core.event.AbstractEvent",
+        "io.scalac.mesmer.core.event.AbstractService",
+        "io.scalac.mesmer.core.event.ActorEvent$TagsSet",
+        "io.scalac.mesmer.core.event.ActorEvent",
+        "io.scalac.mesmer.core.event.EventBus$",
+        "io.scalac.mesmer.core.event.EventBus",
+        "io.scalac.mesmer.core.event.PersistenceEvent",
+        "io.scalac.mesmer.core.event.ReceptionistBasedEventBus$Subscribers",
+        "io.scalac.mesmer.core.event.ReceptionistBasedEventBus$",
+        "io.scalac.mesmer.core.event.ReceptionistBasedEventBus",
+        "io.scalac.mesmer.core.event.Service$",
+        "io.scalac.mesmer.core.event.Service$$anon$1",
+        "io.scalac.mesmer.core.event.Service",
+        "io.scalac.mesmer.core.event.StreamEvent$StreamInterpreterStats",
+        "io.scalac.mesmer.core.event.StreamEvent",
+        "io.scalac.mesmer.core.util.ClassicActorSystemOps$",
+        "io.scalac.mesmer.core.util.ClassicActorSystemOps$ActorSystemOps$",
+        "io.scalac.mesmer.core.util.Interval$",
+        "io.scalac.mesmer.core.util.Interval",
+        "io.scalac.mesmer.core.util.MutableTypedMap$",
+        "io.scalac.mesmer.core.util.MutableTypedMap",
+        "io.scalac.mesmer.core.util.Retry$",
+        "io.scalac.mesmer.core.util.stream$",
+        "io.scalac.mesmer.core.util.TypedActorSystemOps$",
+        "io.scalac.mesmer.core.config.ConfigurationUtils$",
+        "io.scalac.mesmer.core.config.ConfigurationUtils$ConfigOps$",
+        "io.scalac.mesmer.core.invoke.Lookup",
+        "io.scalac.mesmer.core.model.ActorRefTags$ActorRefNonEmptyTags",
+        "io.scalac.mesmer.core.model.ActorRefTags$",
+        "io.scalac.mesmer.core.model.ActorRefTags",
+        "io.scalac.mesmer.core.model.Reporting$Group$",
+        "io.scalac.mesmer.core.model.Reporting$Instance$",
+        "io.scalac.mesmer.core.model.Reporting$Disabled$",
+        "io.scalac.mesmer.core.model.Reporting$",
+        "io.scalac.mesmer.core.model.Reporting",
+        "io.scalac.mesmer.core.model.Tag$StreamName$",
+        "io.scalac.mesmer.core.model.Tag$StreamName",
+        "io.scalac.mesmer.core.model.Tag$SubStreamName$StreamNameWithIsland",
+        "io.scalac.mesmer.core.model.Tag$SubStreamName$",
+        "io.scalac.mesmer.core.model.Tag$All$",
+        "io.scalac.mesmer.core.model.Tag$SubStreamName",
+        "io.scalac.mesmer.core.model.Tag$StageName$",
+        "io.scalac.mesmer.core.model.Tag$StageName",
+        "io.scalac.mesmer.core.model.Tag$StageName$StreamUniqueStageName",
+        "io.scalac.mesmer.core.model.Tag$StreamTag$",
+        "io.scalac.mesmer.core.model.Tag$StreamName$StreamNameImpl",
+        "io.scalac.mesmer.core.model.Tag$TerminatedTag$",
+        "io.scalac.mesmer.core.model.Tag$",
+        "io.scalac.mesmer.core.model.Tag",
+        "io.scalac.mesmer.core.model.stream.StageInfo",
+        "io.scalac.mesmer.core.model.stream.ConnectionStats");
+  }
+
+  public static List<String> combine(List<String>... values) {
+    return Stream.of(values).flatMap(Collection::stream).collect(Collectors.toList());
+  }
+}

--- a/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaHttpInstrumentationModule.java
+++ b/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaHttpInstrumentationModule.java
@@ -67,17 +67,18 @@ public class MesmerAkkaHttpInstrumentationModule extends InstrumentationModule
 
   @Override
   public List<String> getAdditionalHelperClassNames() {
-    return Arrays.asList(
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.RouteContext$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.RouteContext",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.RouteTemplateHolder",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.UpdateHttpRouteWrapper$$anonfun$$nestedInanonfun$apply$1$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.UpdateHttpRouteWrapper",
-        "io.scalac.mesmer.instrumentation.http.impl.RawPathPrefixInterceptor",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.RawPathPrefixImplementation$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.RawPathPrefixImplementation",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.AkkaHttpConnectionsInstrumentation$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.AkkaHttpConnectionsInstrumentation$HttpConnectionInstruments$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.http.AkkaHttpConnectionsInstrumentation");
+    return MesmerAkkaHelpers.combine(
+        MesmerAkkaHelpers.coreHelpers(),
+        List.of(
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.RouteContext$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.RouteTemplateHolder",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.UpdateHttpRouteWrapper$$anonfun$$nestedInanonfun$apply$1$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.UpdateHttpRouteWrapper",
+            "io.scalac.mesmer.instrumentation.http.impl.RawPathPrefixInterceptor",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.RawPathPrefixImplementation$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.RawPathPrefixImplementation",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.AkkaHttpConnectionsInstrumentation$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.AkkaHttpConnectionsInstrumentation$HttpConnectionInstruments$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.http.AkkaHttpConnectionsInstrumentation"));
   }
 }

--- a/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaPersistenceInstrumentationModule.java
+++ b/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaPersistenceInstrumentationModule.java
@@ -33,17 +33,19 @@ public class MesmerAkkaPersistenceInstrumentationModule extends InstrumentationM
 
   @Override
   public List<String> getAdditionalHelperClassNames() {
-    return Arrays.asList(
-        "io.scalac.mesmer.configuration.Config$",
-        "io.scalac.mesmer.configuration.Config",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.Instruments",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.InstrumentsProvider$$anon$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.InstrumentsProvider",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.InstrumentsProvider$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.PersistenceContext",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.PersistenceContextProvider",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.TemplatingPersistenceContextProvider",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.IdentityPersistenceContextProvider");
+    return MesmerAkkaHelpers.combine(
+        MesmerAkkaHelpers.coreHelpers(),
+        Arrays.asList(
+            "io.scalac.mesmer.configuration.Config$",
+            "io.scalac.mesmer.configuration.Config",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.Instruments",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.InstrumentsProvider$$anon$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.InstrumentsProvider",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.InstrumentsProvider$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.PersistenceContext",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.PersistenceContextProvider",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.TemplatingPersistenceContextProvider",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.persistence.impl.IdentityPersistenceContextProvider"));
   }
 
   @Override

--- a/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaStreamInstrumentationModule.java
+++ b/otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaStreamInstrumentationModule.java
@@ -44,30 +44,32 @@ public class MesmerAkkaStreamInstrumentationModule extends InstrumentationModule
 
   @Override
   public List<String> getAdditionalHelperClassNames() {
-    return Arrays.asList(
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ConnectionCounters",
-        "akka.stream.GraphInterpreterOtelPushAdvice$",
-        "akka.stream.GraphInterpreterOtelPullAdvice$",
-        "akka.stream.GraphLogicOtelOps",
-        "akka.stream.GraphLogicOtelOps$",
-        "akka.stream.GraphLogicOtelOps$GraphLogicEnh",
-        "akka.stream.GraphLogicOtelOps$GraphLogicEnh$",
-        "akka.ConnectionOtelOps",
-        "akka.ConnectionOtelOps$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ActorGraphInterpreterOtelDecorator",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ActorGraphInterpreterOtelDecorator$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ActorGraphInterpreterOtelDecorator$$anonfun$addCollectionReceive$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.GraphStageIslandOps",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.GraphStageIslandOps$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.GraphStageIslandOps$TerminalSink$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.PhasedFusingActorMaterializerAdvice$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$$anon$1",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtensionId$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$StreamStatsReceived",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMetrics",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamConfig$",
-        "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamAttributes$");
+    return MesmerAkkaHelpers.combine(
+        MesmerAkkaHelpers.coreHelpers(),
+        Arrays.asList(
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ConnectionCounters",
+            "akka.stream.GraphInterpreterOtelPushAdvice$",
+            "akka.stream.GraphInterpreterOtelPullAdvice$",
+            "akka.stream.GraphLogicOtelOps",
+            "akka.stream.GraphLogicOtelOps$",
+            "akka.stream.GraphLogicOtelOps$GraphLogicEnh",
+            "akka.stream.GraphLogicOtelOps$GraphLogicEnh$",
+            "akka.ConnectionOtelOps",
+            "akka.ConnectionOtelOps$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ActorGraphInterpreterOtelDecorator",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ActorGraphInterpreterOtelDecorator$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.ActorGraphInterpreterOtelDecorator$$anonfun$addCollectionReceive$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.GraphStageIslandOps",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.GraphStageIslandOps$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.GraphStageIslandOps$TerminalSink$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.impl.PhasedFusingActorMaterializerAdvice$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$$anon$1",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtensionId$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMonitorExtension$StreamStatsReceived",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamMetrics",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamConfig$",
+            "io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamAttributes$"));
   }
 }

